### ISSUE TITLE
Can't share to app

### ISSUE THREAD

**Issue by jidanni** [commons-app/apps-android-commons#4593]
Why isn't this intent detected? Other apps detect it.

```
intent:#Intent;action=android.intent.action.SEND;type=application/*;launchFlags=0x1b080000;end 
------------ 
ACTION: android.intent.action.SEND 
MIME: application/* 
URI: intent:#Intent;action=android.intent.action.SEND;type=application/*;launchFlags=0x1b080000;end 
FLAGS: 
FLAG_RECEIVER_REGISTERED_ONLY_BEFORE_BOOT 
FLAG_RECEIVER_FOREGROUND 
FLAG_ACTIVITY_FORWARD_RESULT 
FLAG_ACTIVITY_PREVIOUS_IS_TOP 
FLAG_ACTIVITY_CLEAR_WHEN_TASK_RESET 
EXTRAS: 
1 Class: android.net.Uri$HierarchicalUri 
Key: android.intent.extra.STREAM 
Value: content://media/external/audio/media/266376 

------------ 
MATCHING ACTIVITIES: 
Bluetooth (com.android.bluetooth - com.android.bluetooth.opp.BluetoothOppLauncherActivity) 
Save to Drive (com.google.android.apps.docs - com.google.android.apps.docs.common.shareitem.UploadMenuActivity) 
Maps (com.google.android.apps.maps - com.google.android.maps.MapsActivity) 
Messages (com.google.android.apps.messaging - com.google.android.apps.messaging.ui.conversationlist.ShareIntentActivity) 
Gmail (com.google.android.gm - com.google.android.gm.ComposeActivityGmailExternal) 
Nearby Share (com.google.android.gms - com.google.android.gms.nearby.sharing.ShareSheetActivity) 
BAND (com.nhn.android.band - com.nhn.android.band.ActionSendActivity) 
Term here (jackpal.androidterm - jackpal.androidterm.TermHere) 
LINE (jp.naver.line.android - com.linecorp.line.share.common.view.FullPickerLaunchActivity) 
LINE Keep (jp.naver.line.android - com.linecorp.linekeep.ui.KeepSaveActivity) 
OfflineMaps (net.psyberia.offlinemaps - net.psyberia.offlinemaps.OfflineMapsActivity) 
VLC (org.videolan.vlc - org.videolan.vlc.StartActivity) 
Send to Zoom Buddies (us.zoom.videomeetings - us.zoom.videomeetings.SendFileActivity) 
```

**Comment by nicolas-raoul** [comment]
Does it happen for any file, including pictures? Or only for audio?

**Comment by jidanni** [comment]
Must have been only audio.

**Comment by nicolas-raoul** [comment]
This will be relevant when audio is implemented, so let's redirect this to #4208 for now. Thanks!
